docs/examples/dynamic_provisioning_external_gluster/README.md

```diff
@@ -0,0 +1,572 @@
+	├─sda1        8:1    0    1G  0 part /boot
+	└─sda2        8:2    0   19G  0 part 
+	  ├─cl-root 253:0    0   17G  0 lvm  /
+	  └─cl-swap 253:1    0    2G  0 lvm  [SWAP]
+	sdb           8:16   0  200G  0 disk  <1>
+	sdc           8:32   0  200G  0 disk  <2>
+	sr0          11:0    1 1024M  0 rom  
+```
+<1> 200GB raw device that we will let Heketi manage
+
+<2> 200GB raw device that we will let Heketi manage
+
+
+#### Confirm proper access and communication between the storage nodes.
+
+- Update /etc/hosts with ip and hostname (i.e.  192.168.1.205 gluster25.rhs)
+- Set up password-less SSH communication between all nodes
+
+```
+	# ssh-keygen  (take defaults)
+	# ssh-copy-id -i /root/.ssh/id_rsa.pub root@gluster25.rhs
+```
+***NOTE:*** Repeat this from each node as necessary
+
+
+#### Download and install GlusterFS.
+
+
+```
+	# yum install wget telnet -y
+	# yum install centos-release-gluster -y
+	# yum install epel-release -y
+	# yum install glusterfs-server -y
+```
+
+#### Start glusterd.
+
+```
+	# systemctl start glusterd
+	# systemctl enable glusterd
+```
+
+#### Enable firewall rules.
+
+Add Some Firewall Rules On All Nodes (if you have not disabled firewalld)
+
+```
+	# firewall-cmd --zone=public --add-service=glusterfs --permanent
+	# firewall-cmd --reload
+```
+
+
+#### Setup the Storage Pool.
+
+From one of the GlusterFS storage nodes, peer probe the other members of the pool
+
+```
+	# gluster peer probe gluster26.rhs
+	peer probe: success. 
+
+	# gluster peer status
+	Number of Peers: 1
+
+	Hostname: gluster26.rhs
+	Uuid: 6d7a3149-e163-4de8-9e63-b00e9ceacc59
+	State: Peer in Cluster (Connected)
+
+```
+
+From the other storage node (gluster26.rhs) - run the peer status
+
+```
+	# gluster peer status	
+	Number of Peers: 1
+
+	Hostname: gluster25.rhs
+	Uuid: d4cbbdbf-5dca-46ba-a7d4-2cc5d8bcdb06
+	State: Peer in Cluster (Connected)
+
+```
+
+
+
+### Installing and configuring Heketi Server Node
+
+Heketi is used to manage our gluster cluster storage (adding volumes, removing volumes, etc…). As already stated this example
+uses a stand-alone virtual machine to run Heketi and the client, but this can be installed on one of the existing Gluster Storage Nodes.
+
+
+#### Confirm and validate communication on the Heketi Server Node.
+
+- Similar to the GlusterFS storage nodes, verify /etc/hosts and firewalls
+
+```
+	# firewall-cmd --permanent --zone=public --add-port=8080/tcp
+	# firewall-cmd --reload
+```
+
+#### Install the Heketi Server.
+
+Choose an existing Gluster Storage Node or another Node that can run the Heketi Server with access to the Gluster cluster.
+For this example we are using a non-gluster stand-alone node. For more information see the [Heketi github site](https://github.com/heketi/heketi/wiki/Installation)
+
+```
+	# yum install wget telnet -y
+	# yum install centos-release-gluster -y
+	# yum install epel-release -y
+	# yum install heketi -y
+
+	# heketi version
+	Heketi 2.0.6
+	Please provide configuration file
+```
```

Comment:
Latest version of Heketi is v3.  It could have been better to lay out this example using v3 as it support new features like GID.  I understand you came into v2 as centos sig storage repo has not been updated yet. We will look into that.